src/org/opendatakit/briefcase/export/ExportForms.java

```diff
@@ -117,59 +111,59 @@ public FormStatus get(int rowIndex) {
     return forms.get(rowIndex);
   }
 
-  public void forEach(Consumer<String> callback) {
-    forms.stream().map(ExportForms::getFormId).forEach(callback);
+  public void forEach(Consumer<FormStatus> callback) {
```

Comment:
Can we call this `forEachFormStatus`?

Reply:
Wouldn't that be a little redundant, since the signature explicitly expresses that this is a `forEach` of a list of `FormStatus` objects?

Also, the main purpose of this class is to wrap a list of `FormStatus` objects. I do the same pattern with other wrappers that have `.map()` and `.flatMap` methods. If we really need to add suffixes to clarify it's probably because the wrapper does too much stuff and should be split.